apps/src/templates/ContainedLevel.jsx

```diff
@@ -31,7 +33,37 @@ const ContainedLevel = React.createClass({
       codeStudioLevels.lockContainedLevelAnswers();
     } else {
       // No answers yet, disable Run button until there is an answer
-      $('#runButton').prop('disabled', true);
+      let runButton = $('#runButton');
+      runButton.prop('disabled', true);
+      // Disabled buttons don't trigger mouse events, add a dummy element to
+      // receive the click and trigger the display of the callout.
+      let clickReceiver = $('<div id="clickReceiver"/>');
+      let boundingClientRect = runButton.get(0).getBoundingClientRect();
+      clickReceiver.css({
+        height: boundingClientRect.height + 'px',
+        width: boundingClientRect.width + 'px',
+        position: 'absolute',
+        top: 0,
+      });
+      $('#gameButtons').append(clickReceiver);
+      clickReceiver.bind('click',
+        () => $(window).trigger('attemptedRunButtonClick'));
```

Comment:
Nit: I think you can re-use the [`run_button_pressed` event](https://github.com/code-dot-org/code-dot-org/blob/ef0a7ad16856f7f6f5c6b22200be0552c88e7c5b/apps/src/StudioApp.js#L1691).